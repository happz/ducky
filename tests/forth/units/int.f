0 INVERT      CONSTANT MAX-UINT
0 INVERT 1 RSHIFT   CONSTANT MAX-INT
0 INVERT 1 RSHIFT INVERT  CONSTANT MIN-INT

." MAX-INT  " MAX-INT  DUP . U. CR
." MIN-INT  " MIN-INT  DUP . U. CR
." MAX-UINT " MAX-UINT DUP . U. CR
." MIN-UINT " 0        DUP . U. CR

