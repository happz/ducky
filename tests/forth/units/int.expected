MAX-INT  7FFFFFFF 7FFFFFFF 
MIN-INT  -80000000 80000000 
MAX-UINT -1 FFFFFFFF 
MIN-UINT 0 0 
